applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -20,424 +21,189 @@ def setUp(self):
     def tearDown(self):
         # Code here will be placed AFTER every test in this TestCase.
         pass
-
-    def test_thermal_heat_flux_2D3N(self):
-        test_name = 'test_thermal_heat_flux_2D3N'
-        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', 'test_thermal_heat_flux', test_name))
+        
+    def check_temperature(self, test_directory, test_name, test_node, etalon_value):
```

Comment:
Perhaps we can combine `test_directory` and `test_name` in a single parameter (e.g. `tail_of_test_path`), since the only thing you do with these two parameters is to combine them into a path.